app/views/admin/splashpages/show.html.haml

```diff
@@ -60,6 +60,20 @@
           - else
             No
         %dt
+          Include Venue:
```

Comment:
Those are things that where missed in the show page, right?

They should be in a different commit as they are unrelated to making Splashpage components selected by default :wink: